compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/awt/ComposeBridge.desktop.kt

```diff
@@ -225,7 +235,7 @@ internal class ComposeBridge(
             }
         })
 
-        component.addFocusListener(object : FocusListener {
+        contentComponent.addFocusListener(object : FocusListener {
```

Comment:
Let's not forget to unsubscribe (in a separate PR, or carefully in this), as ComposeBridge is now uncoupled from the component creation.

Reply:
In a separate one